libvirt.go

```diff
@@ -111,6 +111,74 @@ const (
 	DomainXMLFlagMigratable
 )
 
+// scheduler parameters
```

Comment:
Can you add to this comment the name of the function(s) that these are used with?